src/Internet/Url.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ValueObjects\Internet;
+
+use ValueObjects\Exceptions\InvalidUrlException;
+use ValueObjects\ValueObject;
+
+/**
+ * Url
+ *
+ * @author Sergio de Candelario <sergio.decandelario@gmail.com>
```

Comment:
We don't put author in the Phpdocs, take a look at other examples in the code. :D